src/core/Akka.Persistence/Journal/AsyncWriteJournal.cs

```diff
@@ -88,7 +90,7 @@ protected AsyncWriteJournal()
         public abstract Task ReplayMessagesAsync(IActorContext context, string persistenceId, long fromSequenceNr, long toSequenceNr, long max, Action<IPersistentRepresentation> recoveryCallback);
 
         /// <inheritdoc/>
-        public abstract Task<long> ReadHighestSequenceNrAsync(string persistenceId, long fromSequenceNr);
+        public abstract Task<long> ReadHighestSequenceNrAsync(string persistenceId, long fromSequenceNr, CancellationToken cancellationToken);
```

Comment:
[Breaking change] `Akka.Persistence` codes that are being wrapped inside a `CircuitBreaker` will now take a `CancellationToken` as a signal that the operation has been timed out by the `CircuitBreaker`.

Reply:
LGTM